docs/styleguide/PyGuide.md

```diff
@@ -264,263 +239,54 @@ class C(ABC):
     @abstractmethod
     def my_abstract_method(self, ...):
         pass
-    
+
     @classmethod
     @abstractmethod
     def my_abstract_classmethod(cls, ...):
         pass
-    
+
     @staticmethod
     @abstractmethod
     def my_abstract_staticmethod(...):
         pass
-    
+
     @property
     @abstractmethod
     def my_abstract_property(self):
         pass
-    
+
     @my_abstract_property.setter
     @abstractmethod
     def my_abstract_property(self, val):
         pass
-    
+
     @abstractmethod
     def _get_x(self):
         pass
-    
+
     @abstractmethod
     def _set_x(self, val):
         pass
 ```
 
 
-<a id="s3-python-style-rules"></a>
-<a id="3-python-style-rules"></a>
+<a id="s4-python-style-rules"></a>
+<a id="4-python-style-rules"></a>
 <a id="python-style-rules"></a>
-## 3 Python Style Rules 
-
-<a id="s3.1-line-length"></a>
-<a id="31-line-length"></a>
-<a id="line-length"></a>
-### 3.1 Line length 
-
-Maximum line length is *120 characters*.
```

Comment:
Maximum line length is the input parameter for black. So I recommend to fix it in the style guide.

Reply:
Black parameters contains in `pyproject.toml`, and isort and black use it. 
Also `pre-commit run` will reformate code if run tools with another configuration, in result pre-commit job will fall.

I think will be better if collect only rules that will not be check/change automatically.